terraform/modules/psn/main.tf

```diff
@@ -0,0 +1,55 @@
+variable "vpc_id" {
+  type        = "string"
+  description = "The ID of the VPC in which the endpoint will be used."
+}
+
+variable "vpc_endpoint" {
+  type        = "string"
+  description = "The service name, in the form com.amazonaws.region.service for AWS services."
+}
+
+variable "subnet_ids" {
+  type        = "list"
+  description = "The ID of one or more subnets in which to create a network interface for the endpoint."
+}
+
+variable "security_group_ids" {
+  type        = "list"
+  description = "The ID of one or more security groups to associate with the network interface."
+}
+
+resource "aws_vpc_endpoint" "psn_service" {
+  vpc_id            = "${var.vpc_id}"
+  service_name      = "${var.vpc_endpoint}"
+  vpc_endpoint_type = "Interface"
+
+  security_group_ids = ["${var.security_group_ids}"]
```

Comment:
this is already a list and we're putting it inside a list?

Reply:
This was copied from the example Terraform for a VPC endpoint.

Reply:
And it seems to work... hmmm

Reply:
This is how terraform's syntax works. I found it strange at first too.

Reply:
You don't still find it strange? :P

Reply:
It grew on me :D